docs/src/pages/[platform]/connected-components/authenticator/customization/formfields/form-fields.web.mdx

```diff
@@ -17,7 +17,7 @@ have an `order` key that specifies which order it will be displayed on the page.
 
 In the example below the order will be set as `email`, `family_name`, `birthdate`, `preferred_username`, `password` and finally `confirm_password`.
 
-**Note:** Fields that do not have a `order` key will be displayed at the bottom of the Sign Up page. The `order` key can also be combined with other form field updates.
+**Note:** Fields that do not have a `order` key will be displayed at the bottom of the Sign Up page. The `order` key can also be combined with other form field updates. In addition, all attributes listed below have already been inferred through [Zero Configuration](./configuration#sign-up-attributes). If needed, you can explicitly set the [sign up attributes prop](./configuration#sign-up-attributes) to add these to the sign up page.
```

Comment:
If copying the code is a potential poitn of failure for customers shouldn't we be modifying it as well?

Reply:
This explains why we don't need the `signUpAttributes` since the customer should already have them when they setup their project via Zero Configuration. I don't think we should add the signUpAttributes explicitly, but link to it if they need it.

Reply:
But we do we have to use it explicitly ourselves then? https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/%5Bplatform%5D/connected-components/authenticator/customization/formfields/form-fields.web.mdx?plain=1#L30

Reply:
OK, I updated it. We don't do it, because we don't have an `aws-exports` file or a real backend.